docs/ui-extensions-sdk-frontend.md

```diff
@@ -332,6 +332,84 @@ Stops resizing the iframe automatically.
 
 This object provides methods for opening UI dialogs:
 
+### `dialogs.openAlert(options)`
```

Comment:
"dialogs" is a namespace we already have, as in `extension.dialogs.selectMulitpleAssets()`, we just add `extension.dialogs.openAlert()` and others. See the docs for existing dialogs below. For me plural makes more sense since you can open one of many dialogs.

Reply:
:ok